tests/Command/CreateClassCacheCommandTest.php

```diff
@@ -33,6 +35,9 @@ class CreateClassCacheCommandTest extends TestCase
      */
     private $application;
 
+    /**
+     * @group legacy
```

Comment:
shouldn't we move the `@legacy` to the class?

Reply:
Well f*** me, I thought it only works on methods 😅